### PR TITLE
open vote on ‘bump default highWaterMark‘ PR

### PR DESCRIPTION
**Pull request by aduh95**
At the meeting yesterday, it was determined that we need to have a vote on overriding https://github.com/nodejs/node/pull/46608#pullrequestreview-1293747415. Please read the full PR thread for context.

(Given it's a binary vote, I'm choosing to not bother with making the vote secret, even though that would be fairer, because it's still much easier to setup this way, but I'm working on that.)

To vote, please [edit the JSON file](https://github.com/nodejs/TSC/edit/votes/bump-default-highWaterMark/votes/2023-xx-xx-0.json) adding your GitHub handle to the list (ideally in case insensitive alphabetical order) as a key, associated to one of the following value:

- `0` to abstain.
- `1` to vote in favor of overriding the objection (the PR can land once the CI is green, assuming no other objections is raised during the code review process).
- `-1` to vote against overriding the objection (the PR cannot land, the default value doesn't change).

The vote window closes in two weeks, or sooner if every (active/voting) TSC members have voted before that.

/cc @nodejs/tsc

### CODE REVIEW

File: votes/2023-xx-xx-0.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  "refs": [
+    "https://github.com/nodejs/node/pull/46608#pullrequestreview-1293747415"
+  ],
+  "outcome": "TBD",
+  "votes": {
+    "aduh95": 1,
+    "apapirovski": 1,
+    "gireeshpunathil": -1,
+    "ronag": 1,
+    "RaisinTen": 1,
```
Comment (Trott) [inline votes/2023-xx-xx-0.json:12]:
```suggestion
    "RaisinTen": 1,
    "Trott": 0,
```

**Comment by ronag** [comment]
@aduh95 can we close this?

**Comment by aduh95** [comment]
Ah yes, thanks for the ping, I forgot about it, I said I would close it a few days ago. Let me work on that.

**Comment by aduh95** [comment]
12 `+1`s, 1 `-1`, and 5 abstentions. The vote is settle, the objection can be overridden, and this PR can merge after reviews.

**Review by targos — APPROVED** [review]

**Review by gireeshpunathil — APPROVED** [review]

**Review by Trott — APPROVED** [review]

**Pull request merged**